content/en/docs/contribute/intermediate.md

```diff
@@ -379,25 +384,30 @@ After you decide which branch to start your work (or _base it on_, in Git
 terminology), use the following workflow to be sure your work is based on the
 most up-to-date version of that branch.
 
-1.  Fetch both the `upstream` and `origin` remotes. This updates your local
-    notion of what those branches contain, but does not change your local
-    branches at all.
+1.  There are three different copies of the repository when you work locally:
+local, upstream, and remote. Fetch both the origin and upstream remotes. This
```

Comment:
This wording is fine. If you want to given extra clarity to readers, change the first “remote” in this line to “your fork”, or words to that effect.

Reply:
Thanks for catching that. On second look, it should be `origin`.